x/emissions/proto/emissions/v4/tx.proto

```diff
@@ -0,0 +1,411 @@
+syntax = "proto3";
```

Comment:
wonder if this PR changes given https://github.com/allora-network/allora-chain/pull/589#pullrequestreview-2288953783